cfme/infrastructure/pxe.py

```diff
@@ -204,9 +206,11 @@ def exists(self):
         """
         Checks if the PXE server already exists
         """
-        dbs = self.impl()
```

Comment:
im thinking about always passing it in as first positional argument, requiring all other arguments to be keywords - that way a name can be more easily picked, but i suspect some downsides